api_specifications/case-api-0.1.0.yaml

```
@@ -1499,3 +1554,178 @@ components:
+          items:
+            $ref: >-
+              https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/Document
+    JoinCaseV2Response:
+      type: object
+      properties:
+        ResponseInfo:
+          $ref: >-
+            https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ResponseInfo
+        isVerified:
+          type: boolean
+          description: indicates if join a case is successful or not
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)